scripts/Muon/GUI/MuonAnalysis/loadfile/load_file_view.py

```diff
@@ -0,0 +1,180 @@
+
+        self.copy_button.clicked.connect(self.copy_line_edit_to_clipboard)
+
+        self._store_edit_text = False
+        self._stored_edit_text = ""
+        self._cached_text = ""
+
+        # progress tracking if many files being loaded
+        self.progress_bar = None
+        self.progress_widget = None
+
+    def setup_interface_layout(self):
+        self.setObjectName("BrowseFileWidget")
+        self.resize(500, 100)
+
+        self.browse_button = QtWidgets.QPushButton(self)
+
+        size_policy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed)
+        size_policy.setHorizontalStretch(0)
+        size_policy.setVerticalStretch(0)
+        size_policy.setHeightForWidth(self.browse_button.sizePolicy().hasHeightForWidth())
+
+        self.browse_button.setSizePolicy(size_policy)
+        self.browse_button.setMinimumSize(QtCore.QSize(100, 50))
+        self.browse_button.setObjectName("browseButton")
+        self.browse_button.setText("Browse")
+
+        self.file_path_edit = QtWidgets.QLineEdit(self)
+
+        size_policy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
+        size_policy.setHorizontalStretch(0)
+        size_policy.setVerticalStretch(0)
+        size_policy.setHeightForWidth(self.file_path_edit.sizePolicy().hasHeightForWidth())
+
+        self.file_path_edit.setSizePolicy(size_policy)
+        self.file_path_edit.setMinimumSize(QtCore.QSize(200, 50))
+        self.file_path_edit.setToolTip("")
+        self.file_path_edit.setObjectName("filePathEdit")
+
+        self.copy_button = QtWidgets.QToolButton(self)
+        self.copy_button.setMinimumSize(QtCore.QSize(50, 50))
+        self.copy_button.setToolTip("Copy current files to clipboard")
+        self.copy_button.setText("...")
+        self.copy_button.setIcon(QtGui.QIcon(":/copy.png"))
+        self.copy_button.setObjectName("copyButton")
+
+        self.horizontal_layout = QtWidgets.QHBoxLayout(self)
+        self.horizontal_layout.setObjectName("horizontalLayout")
+        self.horizontal_layout.addWidget(self.browse_button)
+        self.horizontal_layout.addWidget(self.file_path_edit)
+        self.horizontal_layout.addWidget(self.copy_button)
+
+        self.setLayout(self.horizontal_layout)
+
+    def on_browse_clicked(self, slot):
+        self.browse_button.clicked.connect(slot)
+
+    def on_file_edit_changed(self, slot):
+        self.file_path_edit.returnPressed.connect(slot)
+
+    def copy_line_edit_to_clipboard(self):
+        cb = QtWidgets.QApplication.clipboard()
+        cb.clear(mode=cb.Clipboard)
+        cb.setText(self.get_file_edit_text(), mode=cb.Clipboard)
+
+    def show_file_browser_and_return_selection(self, file_filter, search_directories, multiple_files=False):
+        default_directory = search_directories[0]
+        if multiple_files:
+            chosen_files, _ = QtWidgets.QFileDialog.getOpenFileNames(self, "Select files", default_directory,
```

Comment:
It was because Mantid uses PyQt4 whereas I used PyQt5 outside of mantid; the return value format of this `QFileDialog` changed between the two versions.